openshift-container-platform-4/policies/AC-Access_Control/component.yaml

```diff
@@ -232,16 +232,23 @@
 - control_key: AC-2 (11)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: partial
+  implementation_status: complete
   narrative:
     - text: |-
-        This control will only be partially satisfied when OpenShift is
-        configured to connect to an Identity Provider (IdP).
-
-        More work is being done to address this control. Engineering
-        progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-210
+       This functionality would fall to third
+       party software such as an Identity Provider (IdP),
+       Active Directory (MSAD), LDAP, etc.
+
+       To configure an IdP, see https://docs.openshift.com/container-platform/latest/authentication/identity_providers/configuring-ldap-identity-provider.html#configuring-ldap-identity-provider
```

Comment:
```suggestion
       To configure an Identity Provider (IdP), refer to the "Authentication" chapter of the OpenShift Container Platform documentation at https://docs.openshift.com/container-platform/latest/authentication/understanding-authentication.html
```

Reply:
The current URL is specific to LDAP. Should the generic "authentication" chapter URL be used instead?